### ISSUE TITLE
[Bug] Search and replace hotkey

### ISSUE THREAD

**Issue by gliviu** [getinsomnia/insomnia#866]
- Insomnia Version: 5.15.0
- Operating System: Windows 10

## Details
I couldn't find any find/replace functionality. CTRL-F implements only searching. Having possibility to replace text would be great addition.
Or maybe I could not find it.

**Comment by gschier** [comment]
Hmm, I think this is because the "Refresh Plugins" hotkey is overriding it. Might have to change that one to something else.

**Comment by michaelsogos** [comment]
I had same problem and solved in this way:
1. The hidden default SHORTCUT for REPLACE is CTRL+SHIFT+R
2. In preferences->keyboard configuration, the default "Filter Sidebar" function is associated to CTRL+SHIFT+R (here override)
3. I changed "Filter Sidebar" to CTRL+H (nothing use same combination)
4. I removed CTRL+SHIFT+R combination from "Filter Sidebar"
5. Now CTRL+H open a search bar on left sidebar, and CTRL+SHIFT+R open "Replace" box

My problem is that REPLACE not works, i mean don't do nothing.
